spotlight-client/src/routerUtils/getUrlForResource.ts

```diff
@@ -50,9 +54,12 @@ function getUrlForResource(opts: GetUrlOptions): string {
       return `/${makeRouteParam(
         opts.params.tenantId
       )}/${DataPortalSlug}/${makeRouteParam(opts.params.metricTypeId)}`;
-    case "narratives":
+    case "narrative list":
```

Comment:
yeah it is very confusing and silly. "collections" used to be the word for what might properly be called "topics" or "tags" in the context of the data portal. then at some point in the latest round of designs it also got adopted as the "display title" of the page that we still refer to internally as "narratives", but they haven't really settled on that as the final title (it might be "narratives", it might be "stories", it might be something else) ... so I was trying to avoid using it in too many places. Typing that out, though, I can see that it doesn't make that much sense! At the very least I should have the title and the URL match up (and if they change later then they both change). But I think the names of components and stuff should be more stable than the display values, to avoid pointless churn, unless they drift so far apart as to become an impediment, so I would leave those as "narrative". Words! 💀 

More fun facts: I don't think there will actually be any relationship between the narratives and collections. They just so happen to completely overlap at the moment but that will not continue to be true. Also, I learned yesterday that the entire data portal feature is getting de-prioritized out of MVP scope, so the "explore" tab is going to come out of the navigation and get put on ice for a while. I may just remove the "collection" data model entirely so it doesn't turn into cruft, since it is not doing anything useful yet anyway.